examples/platform/silabs/BaseApplication.h

```diff
@@ -176,6 +176,8 @@ class BaseApplication
 
 protected:
     CHIP_ERROR Init();
+    CHIP_ERROR BaseInit();
+    virtual CHIP_ERROR AppInit() { return CHIP_NO_ERROR; }
```

Comment:
shoud this be pure virtual instead? This is increase flash due to the vtable having a default implementation.